### DISCUSSION TITLE
color support

### DISCUSSION THREAD

**Discussion by tshepang** [MiSawa/xq#157] [Ideas]
It does not look like there is support for color output? Is this planned?

**Comment by MiSawa** [comment]
Apparently it is in [this list](https://github.com/MiSawa/xq/issues/33#issue-1034096355). Though I don't really have time to work on this project recently, so no ETA unless someone wants to implement it.

**Comment by MiSawa** [comment]
Colored output is supported on #163 (kudos @Humandoodlebug), included in v0.2.43. (except configuration through `$JQ_COLORS`)

**Reply by MiSawa** [reply]
and `$JQ_COLORS` is now supported in #166 / v0.2.44
